user_guide_src/source/outgoing/view_cells.rst

```diff
@@ -94,22 +100,23 @@ At the most basic level, all you need to implement within the class are public p
     }
 
     // app/Cells/alert_message_cell.php
-    <div class="alert alert-<?= $type; ?>">
-        <?= $message; ?>
+    <div class="alert alert-<?= esc($type, 'attr') ?>">
+        <?= esc($message) ?>
     </div>
 
+    // Called in main View:
+    <?= view_cell('AlertMessageCell', 'type=warning, message=Failed.') ?>
+
 .. _generating-cell-via-command:
 
 Generating Cell via Command
 ===========================
 
-.. versionadded:: 4.3.0
-
 You can also create a controlled cell via a built in command from the CLI. The command is ``php spark make:cell``. It takes one argument, the name of the cell to create. The name should be in PascalCase, and the class will be created in the ``app/Cells`` directory. The view file will also be created in the ``app/Cells`` directory.
 
 ::
 
-    > php spark make:cell AlertMessage
+    > php spark make:cell AlertMessageCell
```

Comment:
We should also update all the examples below which declare
```php
class AlertMessage extends Cell
```
to 
```php
class AlertMessageCell extends Cell
```

Reply:
Fixed.